content/Projects/Observability/observatorium.md

```diff
@@ -1,3 +1,57 @@
 # Observatorium
 
-> In progress.
+Observatorium is an observability system design to enable ingestion, storage (short and long term) and querying capabilities for three major observability signals: metrics, logging and tracing. It unifies horizontally scalable, multi-tenant systems like Thanos, Loki and in future Jaeger to deploy them in single stack with consistent APIs. On top of that it's designed to be managed as a service thanks to consistent tenancy, authorization, rate limiting across all three signals.
+
+### Official Documentation
+
+https://observatorium.io
+
+### APIs
+
+TBD
```

Comment:
If these are being left as TBD please can we create a ticket so someone eventually gets round to them?

Reply:
Nice idea!